solutions/fully-configurable/main.tf

```diff
@@ -0,0 +1,253 @@
+#######################################################################################################################
+# Resource Group
+#######################################################################################################################
+
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  existing_resource_group_name = var.existing_resource_group_name
+}
+
+#######################################################################################################################
+# Cloud Logs
+#######################################################################################################################
+
+locals {
+  prefix            = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
```

Comment:
Can you please be consistent with SCC DA, and handle prefix like this -> https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/7713daec73b967316e998bbd6faceb59de967947/solutions/fully-configurable/main.tf#L30

That way you don't have to repeate the `try()` function everywhere in the code